content/about/notes/1.2/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Istio 1.2
+publishdate: 2019-06-18
+icon: notes
+release: 1.2
+---
+
+We’re proud to release Istio 1.2!
+
+We’ve spent the last 3 months making some significant improvements to the overall product,
+with fixes and features from the Istio community. Special thanks to all of our end-users for
+providing feedback, feature requests, and testing the release candidates at various scales.
+
+This release note describes what’s different between Istio 1.1.9 and Istio 1.2.
```

Comment:
Is this true? Aren't we carrying forward from 1.1.0 too?

Reply:
Maybe also add a comment about getting on and sticking to our 3 month cadence which is an achievement.

Reply:
it is how we do release note in the past... many users will be moving from latest 1.1.x to 1.2.

Reply:
We can also point to feature promotions which were achieved through better testing and production experience:


Feature | 1.1 Maturity | 1.2 Maturity
-- | -- | --
SNI (multiple certs) at ingress | Alpha | Stable
Sidecar API | Alpha | Alpha API, Beta runtime
Distributed Tracing | Beta | Stable
Service Tracing | Beta | Stable
Authorization (RBAC) | Alpha | Beta
Cert management on Ingress Gateway | Alpha | Beta
Basic Configuration Resource Validation | Alpha | Beta
Configuration Processing with Galley | Alpha | Beta

Reply:
added, pinged you ad @douglas-reid about service tracing.  can't findn a link to it

Reply:
no one knows service tracing, removing